client/ayon_core/hosts/houdini/plugins/publish/validate_cop_output_node.py

```diff
@@ -26,28 +25,16 @@ def process(self, instance):
         invalid = self.get_invalid(instance)
         if invalid:
             raise PublishValidationError(
-                ("Output node(s) `{}` are incorrect. "
+                ("Output node '{}' is incorrect. "
                  "See plug-in log for details.").format(invalid),
                 title=self.label
             )
```

Comment:
and this be: 
```suggestion
            raise PublishValidationError(
                title=self.label,
                description=(
                    "### Invalid COP output node\n\n"
                    "The output node path for the instance '{}' must "
                    "be set to a valid COP node path. See the log for "
                    "more details.".format(invalid)
                )
            )
```

Reply:
The description should not be used to display instance specific data because then by default you will only see if for one of the instances, not all. Description should hence try to be 'static' for the plug-in.